pulsar/producer_test.go

```diff
@@ -1210,9 +1212,164 @@ func TestTopicTermination(t *testing.T) {
 			return
 		case <-afterCh:
 			assert.Fail(t, "Time is up. Topic should have been terminated by now")
+			return
+		}
+	}
+}
+
+/*
+func TestTopicNotFound(t *testing.T) {
```

Comment:
Since it's difficult to test it, and it may introduce flaky here. The change seems clear to me. I think we can skip these tests for the PR.

Reply:
Useless  test cases are deleted now.